sdk/tables/azure-data-tables/inc/azure/data/tables/tables_clients.hpp

```diff
@@ -3,28 +3,13 @@
 
 #pragma once
 
-#include <azure/core/context.hpp>
-#include <azure/core/credentials/credentials.hpp>
-#include <azure/core/http/http.hpp>
-#include <azure/core/http/policies/policy.hpp>
+#include <azure/core.hpp>
```

Comment:
Include what you are using. All-inclusive headers are for samples and for customers. We should only be including what we use, to minimize compile time and intelisense toll we put on customers.

Reply:
Did you address this one?

Reply:
yep, just did , dis comment got hidden and i missed it

Reply:
Thanks!

Reply:
I still see it. Left another comment to remove. Given that you include many other files below, this line probably can be just dropped.